.github/contributing.md

```diff
@@ -24,11 +24,17 @@ These are general guidelines on the flow from design to implementation
 5. Github pull requests implement the code agreed upon in the GitHub issues
 
 ## Code development
+Following the installation steps (including installing docker) takes around 20-35 minutes. 
 
+Make sure you have git installed (Instructions for git installation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html if not already installed). 
```

Comment:
Recommend Markdown link for styling: `([Instructions for git installation](https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html))`

Reply:
will change the style